server/cluster/cluster_worker.go

```diff
@@ -216,10 +226,17 @@ func (c *RaftCluster) HandleBatchReportSplit(request *pdpb.ReportBatchSplitReque
 	}
 	last := len(regions) - 1
 	originRegion := proto.Clone(regions[last]).(*metapb.Region)
-	hrm = core.RegionsToHexMeta(regions[:last])
 	log.Info("region batch split, generate new regions",
 		zap.Uint64("region-id", originRegion.GetId()),
 		zap.Stringer("origin", hrm),
 		zap.Int("total", last))
+	for _, region := range hrm {
+		regionInfo := core.NewRegionInfo(region, nil)
+		if err = c.HandleRegionHeartbeat(regionInfo); err != nil {
```

Comment:
I think we need a lightweight way to handle them.

Reply:
yes, it will simplify the process and remove some update:
1. hotstat should not update.
2. regionGuide will replace with conf or new.